front/src/providers/details/KeyProductionData.tsx

```diff
@@ -0,0 +1,35 @@
+import { Button } from '@codegouvfr/react-dsfr/Button';
+import Title2 from '../../titles/Title2';
+import { backendClient } from '../../clients/back-client';
+import { OidcClient } from '../../types';
+
+interface KeyProductionDataProps {
+  oidcClientForm: OidcClient;
+}
+
+export const KeyProductionData: React.FC<KeyProductionDataProps> = ({
+  oidcClientForm,
+}) => {
+  return (
+    <div className="fr-mb-10v">
+      <Title2 title="Clés des données de production" id="save" />
+      <p>
+        Pour accéder aux données de production, il vous faut une habilitation
+        juridique sur Datapass. Vous recevrez vos clés de production dès que
+        l’habilitation sera validée (XX jours en moyenne).
```

Comment:
"XX jours" Don't forget to change that one day

Reply:
Do we know the average number of days?